### PR TITLE
Create ui-c8y-1018-503-66-the-app-logo-height-was-not-correctly-set-if-a-new.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-52036](https://cumulocity.atlassian.net/browse/MTM-52036)
Original PR: [5843](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5843)

### CODE REVIEW

File: content/change-logs/ui-c8y-1018-503-66-the-app-logo-height-was-not-correctly-set-if-a-new.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: The app logo height was not correctly set if a new branding is created [GRAFT][release/y2024] (#5843)
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1018-503-66-the-app-logo-height-was-not-correctly-set-if-a-new.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: App logo height fixed when creating new branding
```

File: content/change-logs/ui-c8y-1018-503-66-the-app-logo-height-was-not-correctly-set-if-a-new.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-0UgqXH1Ys
+    label: Administration
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-52036
+version: 1018.503.66
+---
+The app logo height was not correctly set if a new branding is created [GRAFT][release/y2024] (#5843)
```
Comment (gen-App-Enablement) [inline content/change-logs/ui-c8y-1018-503-66-the-app-logo-height-was-not-correctly-set-if-a-new.md:17]: The Claude AI suggested to use the following body:
```suggestion
Fixed an issue where the app logo height was not set correctly when creating a new branding. The app logo height will now be set properly when generating a new branding.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by janhommes — APPROVED** [review]

File: content/change-logs/ui-c8y-1018-503-66-the-app-logo-height-was-not-correctly-set-if-a-new.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: App logo height default value corrected when creating new branding
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1018-503-66-the-app-logo-height-was-not-correctly-set-if-a-new.md:3]:
```suggestion
title: App logo height set properly when creating new branding
```

**Pull request merged**